### PR TITLE
Rewrite of package manager to change workflow

### PR DESCRIPTION
**Pull request by alexkilgour**
Major rewrite of the package manager:

* Remove extended packages
    * All packages now live in one of the toolkits
    * Packages across multiple toolkits live in Global
    * Global toolkit packages contain branding for other toolkits
* Single brand context package
    * Contains configuration for each brand
    * Context now lives in a different location to other packages
* Refactor of the app flow to handle new functionality

Still to do:
* New major version
* retire `util-package-extender`

New functionality is detailed in the `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +89,7 @@ The default configuration options provided can be overriden and/or extended in o
 
 #### Repository
 
-By providing a `package-manager.json` file in the root of your repository, in the same location as your `package.json` file. Configuration options defined here will extend/override those in the default (with the exception of the `toolkitsDirectory` and `packagesDirectory` options), and are inherited by _all_ toolkits.
+By providing a `package-manager.json` file in the **`toolkits`** folder of your repository. Configuration options defined here will extend/override those in the default (with the exception of the `toolkitsDirectory` and `packagesDirectory` options), and are inherited by _all_ toolkits.
```
Comment (davidpauljunior) [inline README.md:92]: Does specifying the root make it clearer?
"By providing a `package-manager.json` file at the root of the **`toolkits`** folder" of your repository.

File: README.md [hunk 2]
```diff
@@ -122,9 +125,10 @@ The following example would allow a folder with the name `js` that contains file
 ```
 
 #### CSSDirectoryStructure
-This option allows you to specify a custom CSS folder structure. This is used in the [package creation](#package-creation) step to generate a sub-folder structure within a specified folder, to assist in quickly spinning up a new package. It is not used in the validation or publication steps.
 
-The following shows an example folder structure, taken from the [frontend-global-toolkit](https://github.com/springernature/frontend-global-toolkit):
+This option allows you to specify a custom CSS folder structure. This is used in the [package creation](#package-creation) step to generate a sub-folder structure within a specified folder, to assist in quickly spinning up a new package. It is not used in the validation or publication steps (more on those steps later).
```
Comment (davidpauljunior) [inline README.md:129]: "more on those steps later" - Do we need this? Could we anchor link validation and publication?

File: README.md [hunk 3]
```diff
@@ -298,10 +325,10 @@ It is also required that the package `changelog` has been updated, and appears i
 
 #### travis
```
Comment (davidpauljunior) [inline README.md:326]:
```suggestion
#### Travis
```

File: __mocks__/glob-results.js [hunk 4]
```diff
@@ -127,11 +140,40 @@ const packageFiles = () => {
...
+			'packages/package/passContext/brandA/folder2/file.js',
+			'packages/package/passContext/brandA/folder2/file.spec.js',
+			'packages/package/passContext/brandA/folder2/file.json',
+			'packages/package/passContext/brandA/folder2/subfolder',
+			'packages/package/passContext/brandA/folder2/subfolder/file.js'
+		]
+	};
+};
+
+const folderFiles = () => {
```
Comment (davidpauljunior) [inline __mocks__/glob-results.js:162]: Is there a reason that these are functions that return objects, rather than just objects?

Comment (alexkilgour) [reply __mocks__/glob-results.js:162]: Absolutely no reason at all. Not sure why I've done that!

**Pull request merged**